client/.eslintrc.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
     'no-use-before-define': 0,
     'react/no-multi-comp': 0,
     'import/no-mutable-exports': 0,
-    'react/jsx-props-no-spreading': 'warn'
+    'react/jsx-props-no-spreading': 'warn',
+    'no-console': 'warn'
```

Comment:
I will remove these but for now, while we have test data, I think it makes sense to keep for now.

Reply:
Can you make an issue to track that? Seems like an easy thing to forget about.